src/transforms.jl

```diff
@@ -170,16 +170,17 @@ reapply(transform::StatelessFeatureTransform, table, cache) =
 # ------------------
 
 function applyfeat(transform::ColwiseFeatureTransform, feat, prep)
-  # basic checks
-  for assertion in assertions(transform)
-    assertion(feat)
-  end
-
   # retrieve column names and values
   cols   = Tables.columns(feat)
   names  = Tables.columnnames(cols)
   snames = choose(transform.colspec, names)
 
+  # basic checks 
+  for assertion in assertions(transform) 
+    # Perform the assertion on the selected columns of the input table
+    assertion(feat[:, snames]) 
```

Comment:
This syntax is specific to DataFrame. You can replace it with `feat |> Select(snames)` to get a new table with only the selected columns.

Reply:
Yeah, Now it working fine, Thanks!
```jl
julia> (a=rand(Int,3), b=rand(3)) |> ZScore(:b)
(a = [983918827768240371, 3725987254810195682, -5701988653384801394], b = [0.2585094601557376, 0.8453629612542206, -1.1038724214099582])
```